packages/browsers/src/Cache.ts

```diff
@@ -159,6 +182,27 @@ export class Cache {
         });
     });
   }
+
+  computeExecutablePath(options: ComputeExecutablePathOptions): string {
```

Comment:
Does this mean we have a duplication of this functionality on the Cache and in `launch.ts`?
Could `launch.ts` reexport this if it were static method?

Reply:
ah yes, forgot to update that part.